### ISSUE TITLE
make a script to post to nostr

### ISSUE THREAD

**Issue by bitkarrot** [lightningames/astroblog-nostr#2]
write a script to post the latest .md file commit to nostr. 
will need ability to configure nostr credentials and add nostr relays

**Comment by bitkarrot** [comment]
I added a basic framework for a python script in script.py 

If you fork this repo, you'll need to add the secret to your fork for testing out nostr posting.

**Comment by bitkarrot** [comment]
might be helpful to use this library for posting to nostr 

Aionostr library: https://github.com/davestgermain/aionostr

**Comment by BrutusBondBTC** [comment]
Do I need to pay testnet sats to use the pleb.dev relay?

**Comment by bitkarrot** [comment]
yes you can get testnet sats by visiting https://htlc.me and use it for the relay at https://testnet.plebnet.dev/nostrrelay/1

**Comment by BrutusBondBTC** [comment]
Do I need to pay testnet sats to use the pleb.dev relay?

Thanks. I did it, but I didn't write down my testnet wallet key. No biggie, I guess.

**Comment by bitkarrot** [comment]
Its okay. if you lose it, you can just create a new testnet wallet. 

Yes you do need to pay the testnet sats :)

**Comment by bitkarrot** [comment]
Closing as part of repository cleanup; this repo is being archived.
